### PR TITLE
Use contentEditable for editable table cells

### PR DESCRIPTION
**Pull request by tuckerconnelly**
This makes it so editable text cells in Table wrap, instead of getting cut off.

screen shot 2016-09-09 at 2 24 37 pm

### CODE REVIEW

**Comment by acornejo** [comment]
I was interested in this, it was hard to read because your commit touches 45 files, I don't think you need to include the lib/* files as part of your PR.

**Comment by coljung** [comment]
I think you editing all the lib/* files make this merge not good. If you are updating table cell in the Table component, those should be the only files in your PR.

**Comment by javivelasco** [comment]
I agree. Please update the changes removing the build so it can be more clear to review. In any case a new build should be created after the release / merge with master!

**Comment by tuckerconnelly** [comment]
Okay updated the PR. Some revert conflict weirdness happened so one lib file is still in there, but should be easier to read now.

Why is the lib even committed? Isn't .npmignore enough?

File: components/table/TableRow.js [hunk 1]
```diff
@@ -62,14 +75,26 @@ const factory = (Checkbox) => {
...
+      if (inputType === 'text') {
+        return (
+          <div
+            contentEditable
+            suppressContentEditableWarning
+            onInput={this.handleInputChange.bind(null, index, key, inputType)}>
+            {inputValue}
+          </div>
+        );
+      } else {
```
Comment (coljung) [inline components/table/TableRow.js]: I dont think the else > return is necessary in this case. See: http://eslint.org/docs/rules/no-else-return

Comment (javivelasco) [inline components/table/TableRow.js]: +1 Apart from that great!

Comment (tuckerconnelly) [inline components/table/TableRow.js]: Aight it's in

**Pull request merged**